addon/content/reducer.js

```diff
@@ -33,6 +33,10 @@ const initialSummary = {
   subject: "",
 };
 
+// conversations sometimes stores a tag as
+// { color, id, name } but the internal TB
+// representation is { color, key, tag }.
+// This function converts to the conversations representation
```

Comment:
This function (`modifyOnlyMsg`) handles only updating a single message in the reducer. I'm not sure where this comment is meant to be attached to.

Reply:
My bad...that leaked from my PR for the tag refactoring...